server/src-lib/Hasura/Server/Init.hs

```diff
@@ -391,6 +392,13 @@ pgTimeoutEnv =
   , "Each connection's idle time before it is closed (default: 180 sec)"
   )
 
+pgStaleTimeoutEnv :: (String, String)
```

Comment:
We could call it `HASURA_GRAPHQL_PG_LIFETIME` or similar as both pgBouncer and pgpool use the word `lifetime` for signifying this. But, I am okay with `HASURA_GRAPHQL_PG_STALE_TIMEOUT` as well.

https://www.pgbouncer.org/config.html
https://www.pgpool.net/docs/latest/en/html/runtime-config-connection-pooling.html

Reply:
Thanks, yep it seems good to reuse that language.